internal/services/containers/kubernetes_fleet_member_test.go

```diff
@@ -0,0 +1,179 @@
+package containers_test
```

Comment:
Can we add a test case to demonstrate the multi-cluster member case, albeit the update test case?